core/src/main/java/jenkins/model/queue/QueueItem.java

```diff
@@ -73,6 +73,15 @@ default boolean hasCancelPermission() {
     @CheckForNull
     @Override
     default String getDisplayName() {
+        // TODO review usage of this method and replace with getFullDisplayName() where appropriate
+        return getTask().getFullDisplayName();
```

Comment:
Not sure I understand the comment. Did you mean
```suggestion
        return getTask().getDisplayName();
```
? And should `ResourceActivity` implement `ModelObject`?

Reply:
The name of the method implies this would be the natural implementation, but since it actually uses the full name, these could cause some UI inconsistencies if changed without reviewing all current usages.

Reply:
> And should ResourceActivity implement ModelObject?

I had missed this interface, I guess it could.

Reply:
So do I understand the comment to mean that you propose switching `QueueItem.displayName` to use `Task.displayName` but looking up existing callers (probably in Jelly) and switching them from `${i.displayName}` to `${i.fullDisplayName}` where appropriate? If so, that was not clear to me from the comment text.